.github/workflows/ci-build.yaml

```
@@ -32,7 +32,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.22'
+          go-version: '1.24.4'
```

```suggestion
          go-version: '1.24'
```
Can we drop z value so that it will use latest available 1.24.z automatically